### DISCUSSION TITLE
Incremental training of Spacy NER.

### DISCUSSION THREAD

**Discussion by baivabdash** [explosion/spaCy#9414] [Help: Coding & Implementations] [answered]
Hi, 
I have trained a NER model on a bunch of data and have deployed the model. Now after a month, I will be having new data. So in that case can I do incremental training of the model only on the new data or would I have to retrain the model with entire (old and new) data ? 
P.S: I am having text from the pages of pdf as training data.

**Comment by polm** [comment, marked as answer]
We generally recommend training from scratch to avoid catastrophic forgetting.

**Reply by baivabdash** [reply]
Okay, so there is no way that I can do incremental training using spacy. Thanks for your quick response.

**Reply by polm** [reply]
That's not correct - it's possible, but in most cases retraining works better. Please look at [the related FAQ entries](https://github.com/explosion/spaCy/discussions/8226#:~:text=my%20retrained%20model%20forgot%20pretrained%20entities).
